cpg_pipes/providers/status.py

```diff
@@ -59,3 +63,39 @@ def add_updaters_jobs(
         """
         Add Hail Batch jobs that update the analysis status.
         """
+
+    def slack_env(self, j: Job):
```

Comment:
Should this be part of the `StatusReporter` interface? I would have expected the Slack reporter to be one concrete subclass of a `StatusReporter`. Can we come up with a more generic interface that would fit e.g. both Slack and SMDB updates?